modules/ui/sections/map_daterange.js

```diff
@@ -30,6 +27,9 @@ export function uiSectionDateRange(context) {
     // see modules/renderer/features.js checkDateFilter() which applies the filters
     // see modules/renderer/features.js update() which updates URL params
 
+    const MIN_YEAR = context.minYear();
+    const MAX_YEAR = context.maxYear();
```

Comment:
This change is unrelated to the validator rule. I had originally planned to make the validator enforce this minimum and maximum year on the `start_date` and `end_date` tags before realizing that the range is primarily for display purposes.

Reply:
I ended up removing this change from the PR, since it’s based on the date filter feature that hasn’t been cherry-picked yet.